### PR TITLE
[Feat] [Translations] [Uzbek]: uz/advanced/structuring

### PR DESCRIPTION
**Pull request by sad1go0**

### CODE REVIEW

File: site/docs/uz/advanced/structuring.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+prev: ./middleware.md
+next: ./scaling.md
+---
+
+# Kengaytirish I: Katta kod bazasi
+
+Sizning botingiz murakkablashib borishi bilan siz dastur kodlari bazasini qanday tuzish masalasiga duch kelasiz.
+Tabiiyki, siz uni fayllarga bo'lishingiz mumkin.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:9]: fayllarga bo'lishingiz => turli fayllarga ajratishingiz

File: site/docs/uz/advanced/structuring.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+prev: ./middleware.md
+next: ./scaling.md
+---
+
+# Kengaytirish I: Katta kod bazasi
+
+Sizning botingiz murakkablashib borishi bilan siz dastur kodlari bazasini qanday tuzish masalasiga duch kelasiz.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:8]: bazasini => strukturasini

File: site/docs/uz/advanced/structuring.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+Tabiiyki, siz uni fayllarga bo'lishingiz mumkin.
+
+## Yechim
+
+> grammY hali juda yosh va hali DI konteynerlari bilan rasmiy integratsiyani ta'minlamaydi.
+> Biz buni qo'llab-quvvatlasak, xabardor bo'lish uchun [@grammyjs_news](https://t.me/grammyjs_news) ga obuna bo'ling.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:14]: buni qo'llab-quvvatlasak, xabardor => integratsiyani amalga oshirganimizda xabardor

File: site/docs/uz/advanced/structuring.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+Siz o'zingizning kodingizni xohlaganingizcha tuzishingiz mumkin va barchaga mos keladigan yagona yechim yo'q.
+Aytish joizki, kodingizni tuzishning sodda va tasdiqlangan strategiyasi quyidagilardir.
+
+1. Semantik jihatdan bir xil faylda (yoki kod hajmiga, jildga qarab) bir-biriga tegishli narsalarni birlashtiring.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:19]: remove "bir xil faylda"

Comment (mcpeblocker) [reply site/docs/uz/advanced/structuring.md:19]: tegishli => mos keladigan

Comment (mcpeblocker) [reply site/docs/uz/advanced/structuring.md:19]: birlashtiring => bir faylda birlashtiring

File: site/docs/uz/advanced/structuring.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+   Ushbu qismlarning har biri belgilangan xabarlarni boshqaradigan middlewareni ko'rsatadi.
+2. Botni yozish orqali barcha middlewareni birlashtiradigan bot misolini yozing.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:21]: bot misolini yozing => bot nusxasini yarating

Comment (mcpeblocker) [reply site/docs/uz/advanced/structuring.md:21]: Botni yozish orqali => Botga o'rnatish orqali

File: site/docs/uz/advanced/structuring.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+3. (Ixtiyoriy.) Yangilanishlarni markaziy tarzda oldindan filtrlang va yangilanishlarni faqat toʻgʻri yoʻl bilan yuboring.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:22]: Yangilanishlarni => Updatelarni

File: site/docs/uz/advanced/structuring.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+   Buning uchun `bot.route` ([API Reference](https://doc.deno.land/https://deno.land/x/grammy/mod.ts/~/Composer#route)) yoki muqobil ravishda [router plugini](./plugins/router.md)dan foydalanishingiz mumkin.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:23]: Reference => Ma'lumotnomasi

File: site/docs/uz/advanced/structuring.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+Yuqoridagi strategiyani amalga oshiradigan ishga tushiriladigan misolni [Example Bot repository](https://github.com/grammyjs/examples/tree/main/scaling)sida topishingiz mumkin.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:25]: amalga oshiradigan ishga tushiriladigan => qo'llagan holda ishlaydigan

Comment (mcpeblocker) [reply site/docs/uz/advanced/structuring.md:25]: repository => repozitoriyasi

File: site/docs/uz/advanced/structuring.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+## Struktura misoli
+
+TODO ro'yxatini boshqaradigan oddiy bot uchun siz ushbu strukturani ko'rishingiz mumkin.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:29]: ko'rishingiz => tasavvur qilishingiz

Comment (mcpeblocker) [reply site/docs/uz/advanced/structuring.md:29]: oddiy bot => sodda bot

File: site/docs/uz/advanced/structuring.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+src/
+├── bot.ts
+└── todo/
+    ├── item.ts
+    └── list.ts
+```
+
+`item.ts` shunchaki TODO elementlari haqidagi ba'zi narsalarni belgilaydi va bu kod qismlari `list.ts` da qo'llaniladi.
+
+`list.ts` da ushbu misolni yozishingiz mumkin qilasiz:
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:41]: `list.ts`da esa quyidagi kabi yozishingiz mumkin.

File: site/docs/uz/advanced/structuring.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+
+```ts
+export const lists = new Composer();
+
+// Middlewareni odatdagidek boshqaradigan ba'zi handlerlarni bu erda ro'yxatdan o'tkazing.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:46]: // Odatdagi kabi ba'zi bir xabar boshqaruvi middlewarelarini yozing.

File: site/docs/uz/advanced/structuring.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+lists.on('message', ctx => { ... });
+```
+
+Ixtiyoriy, siz modul ichida bo'lgan barcha xatolarni boshqarish uchun "[error boundary](./guide/errors.md#error-boundaries)"dan foydalanishingiz mumkin.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:50]: Ixtiyoriy, siz => Ixtiyoriy ravishda siz

Comment (mcpeblocker) [reply site/docs/uz/advanced/structuring.md:50]: ichida bo'lgan => ichida kelib chiqadigan

File: site/docs/uz/advanced/structuring.md [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+
+Endi `bot.ts` da ushbu modulni quyidagi tarzda yozishingiz mumkin:
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:52]: ushbu modulni quyidagi => quyidagi

File: site/docs/uz/advanced/structuring.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+
+```ts
+import { lists } from "./todo/list";
+
+const bot = new Bot("<token>");
+
+bot.use(lists);
+// ... bu yerda `todo` kabi modullar ko'proq bo'lishi mumkin
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:60]: kabi modullar ko'proq => kabi ko'proq modullar

File: site/docs/uz/advanced/structuring.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+
+bot.start();
+```
+
+Optionally, you can use the [router plugin](./plugins/router.md) or to bundle up the different modules, if you're able to determine which middleware is responsible upfront.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:65]: Ixtiyoriy ravishda siz turli xil modullarni bir joyga to'plash uchun [router plugin](./plugins/router.md)ni ishlatishingiz mumkin.

File: site/docs/uz/advanced/structuring.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+
+Biroq, esda tutingki, botingizni qanday tuzishning aniq usulini umumiy tarzda tushuntirish mushkulroq.
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:67]: mushkulroq => ancha mushkul

File: site/docs/uz/advanced/structuring.md [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+Har doimgidek, dasturiy ta'minotda, buni eng mantiqiy tarzda bajaring :wink:
```
Comment (mcpeblocker) [inline site/docs/uz/advanced/structuring.md:68]: `Har doimgidek, buni eng mantiqli tarzda bajarishingiz kerak :wink:`

**Review by mcpeblocker — CHANGES_REQUESTED** [review]

File: site/docs/uz/advanced/structuring.md (hunk 4 above)
Comment (KnorpelSenf) [reply site/docs/uz/advanced/structuring.md:19]: Useful hint for you guys: you can click the suggestion formatting button to add inline suggestions:
```suggestion
1. Semantik jihatdan bir xil faylda (yoki kod hajmiga, jildga qarab) bir-biriga tegishli narsalarni birlashtiring. Yay!
```
(Don't apply this)

These suggestions can directly be committed from the web interface. However, if you go to the Files tab, you can add several suggestions to one batch and commit them all at once. That way, you don't end up with hundreds of commits :)

Comment (mcpeblocker) [reply site/docs/uz/advanced/structuring.md:19]: Thanks for useful tip :)

**Review by mcpeblocker — APPROVED** [review]

**Pull request merged**